Python Part/lib/fnn.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from lib.sigmoid import sigmoid
+from utils.report_generator import generate_report
+
+
+class fnn_layers:
+    def __init__(self, input_neurons, output_neurons):
+        self.input_neurons = input_neurons
+        self.output_neurons = output_neurons
+
+        np.random.seed(1000)
+
+        print("\n\n")
+        print("built a neural network with %d input, %d output" % (
+            self.input_neurons, self.output_neurons))
+        print("\n\n")
+        self.__output_weights = np.random.uniform(-1, 1, size=(
+            self.output_neurons, input_neurons))
+
+    def evaluate_model(self, epoch_size, alpha, images_train, labels_train, images_test, labels_test, model_name):
+        accuracy_train = []
+        accuracy_test = []
+        for epoch in range(epoch_size):
+            print(f"================----- epoch: {epoch} -----================")
+            accuracy_train.append(self.backward_without_hidden(alpha=alpha, labels=labels_train, images=images_train))
+            accuracy_test.append(self.forward_without_hidden(images=images_test, labels=labels_test))
+
+        generate_report(
+            accuracy_train=accuracy_train, accuracy_test=accuracy_test, epoch_size=epoch_size,
+            training_sample_size=images_train.shape[0], testing_sample_size=images_test.shape[0], alpha=alpha,
+            input=self.input_neurons, output=self.output_neurons, model_name=model_name, hidden=None,
+            hidden_weights=None, output_weights=self.__output_weights
+
+        )
+
+    def forward_without_hidden(self, images, labels):
```

Comment:
function name should be simply `forward`

Reply:
K